examples/smc.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
Can you add a little docstring at the top of this file, similar to those in other examples. Something like
```py
"""
This file demonstrates how to use the SMCFilter algorithm with a simple model of a noisy harmonic oscillator.
"""
```